[KOGITO-8401] Add a brief introduction to Kogito SW capabilities in the home page (#272)
kiegroup/kogito-docs: serverlessworkflow/modules/ROOT/pages/index.adoc

= {page-component-title}

> {product_name} is a tool to build cloud-native workflow applications. {product_name} can be used to do services and events orchestration and choreografy. Currently, with {product_name} you can integrate with services and events in your architecture using:

Reviewer: what choreografy means in this context?

Author: Here: https://en.wikipedia.org/wiki/Service_choreography

Reviewer: I see, thanks.

> {product_name} is a tool to build cloud-native workflow applications. {product_name} can be used to do services and events orchestration and choreografy. Currently, with {product_name} you can integrate with services and events in your architecture using:

Reviewer 2: We can avoid the passive voice. 
```suggestion
{product_name} is a tool to build cloud-native workflow applications. You can use the {product_name} to do the services and events orchestration and choreography. Currently, with {product_name} you can integrate with services and events in your architecture using:
```

> {product_name} is a tool for building cloud-native workflow applications. You can use the {product_name} to do the services and events orchestration and choreography. Currently, with {product_name} you can integrate with services and events in your architecture using:

Reviewer 3: I agree with @radtriste, here we should probably use `{product_name} {context}` as we are referring to a specific part of Kogito.

> 1. **CloudEvents**. Ideal for an Event-Driven architecture where the services are ready to consume and produce events working in more reactive way.

Reviewer 2: ```suggestion
1. **CloudEvents**. Ideal for an Event-Driven architecture where the services are ready to consume and produce events working in a more reactive way.
```

> 2. **Sync or Async REST services invocations via OpenAPI/Async API**. There are options also to directly call a REST service in the architecture or ecosystem. Either async and sync methods are supported depending on your requirements.

Reviewer 2: ```suggestion
2. **Sync or Async REST services invocations via OpenAPI/Async API**. There are options even to directly call a REST service in the architecture or ecosystem. Either async or sync methods are supported depending on your requirements.
```

3. **Internal Service execution or invocation**. {product_name} is also a framework to build Java Workflow applications. You can use it to create custom Java services to run an application process within the same instance.

> Learn how to do start creating, managing, and deploying your workflow applications with the guides below.

Reviewer 2: ```suggestion
You can learn how to create, manage, and deploy your workflow applications with the following guides.
```
